### PR TITLE
SDK updated for version 2017-03-01 of the Traffic Manager API

### PR DESCRIPTION
**Pull request by vladca**
## Description

https://github.com/Azure/azure-rest-api-specs/pull/993/files
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [x] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/generate.cmd
Comment (markcowl) [inline src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/generate.cmd]: Once the rest-api-specs PR is merged, you will need to update this to reference the last commit in that PR, and not the floating reference to 'master'

**Review by markcowl — CHANGES_REQUESTED** [review]
One comment about geenration in generate.cmd.  Once your swagger spec is merged and you have made this change, this looks good.

**Comment by cormacpayne** [comment]
@azuresdki test this please

**Comment by cormacpayne** [comment]
@vladca it looks like you are used AutoRest 0.16.x to generate the code for this PR, which is a few months old. Please generate the code based off of the latest version of AutoRest (https://github.com/Azure/autorest#installing-autorest)

You will also need to update the following items:
- `generate.cmd`
    - [AutoRest version used](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/generate.cmd#L7)
    - [URL of the spec used (should point to the commit that merged the spec in, not the branch)](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/generate.cmd#L9)
- `project.json`
    - [Package version needs to be bumped (`3.0.0-preview` if there are any breaking changes, `2.2.0-preview` otherwise)](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/project.json#L2)
- `AssemblyInfo.cs`
    - [`AssemblyFileVersion` will need to have the same version that was used in `project.json`](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/Properties/AssemblyInfo.cs#L12)

**Comment by vladca** [comment]
Cormac, please merge with version 0.16.0 for this update, I will upgrade to autorest 1.0+ with the next PR. All other comments (generate.cmd, assembly and package versions) were already addressed.

File: src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/Properties/AssemblyInfo.cs
Comment (cormacpayne) [inline src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/Properties/AssemblyInfo.cs]: @vladca please keep `AssemblyVersion` at 2.0.0.

`AssemblyFileVersion` was the only version that should have changed, so it can remain at 2.2.0

Comment (vladca) [inline src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/Properties/AssemblyInfo.cs]: You're right, sorry

**Comment by cormacpayne** [comment]
@vladca please add back the Microsoft copyright header to all files.

LGTM once this has been resolved

**Review by cormacpayne — APPROVED** [review]

**Pull request merged**
